<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1074</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:#fffff</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>1331</width>
     <height>111</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:white;
font: 63 36pt &quot;Segoe UI Variable Text Semibold&quot;;
border-radius:30px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>700</y>
     <width>1371</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
border-radius:30px;</string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="gallery_butt">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>700</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Variable Text Semibold</family>
     <pointsize>24</pointsize>
     <weight>7</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#gallery_butt{background-color: rgb(0, 0, 0);
font: 63 24pt &quot;Segoe UI Variable Text Semibold&quot;;
color: rgb(255, 255, 255);
border-radius:10px;
}
QPushButton#gallery_butt:hover{
background-color: rgb(75, 75, 75);
}</string>
   </property>
   <property name="text">
    <string>Gallery</string>
   </property>
  </widget>
  <widget class="QPushButton" name="participation_butt">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>700</y>
     <width>301</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#participation_butt{background-color: rgb(0, 0, 0);
font: 63 24pt &quot;Segoe UI Variable Text Semibold&quot;;
color: rgb(255, 255, 255);
border-radius:10px;
}
QPushButton#participation_butt:hover{
background-color: rgb(75, 75, 75);
}</string>
   </property>
   <property name="text">
    <string>Participation</string>
   </property>
  </widget>
  <widget class="QPushButton" name="contact_butt">
   <property name="geometry">
    <rect>
     <x>1110</x>
     <y>700</y>
     <width>261</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#contact_butt{font: 63 24pt &quot;Segoe UI Variable Text Semibold&quot;;
border-width: 5;
border-radius: 10;
border-style: solid;
border-color:black;
background-color:black;
color:white;
}
QPushButton#contact_butt:hover{
background-color: rgb(75, 75, 75);
}

</string>
   </property>
   <property name="text">
    <string>Connect </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>-50</x>
     <y>0</y>
     <width>2001</width>
     <height>801</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/Pictures/Screenshots/Screenshot 2024-03-09 201320.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>bg_edited.png</pixmap>
   </property>
  </widget>
  <widget class="QFrame" name="Ignitra">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>351</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-width: 0;
border-radius: 20px;
border-style: solid;
border-color: rgb(0,0,0,100);
color:black
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);


</string>
    </property>
    <property name="text">
     <string>Event Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);


</string>
    </property>
    <property name="text">
     <string>Venue:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);



</string>
    </property>
    <property name="text">
     <string>From:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);


</string>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);


</string>
    </property>
    <property name="text">
     <string>Date:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);


</string>
    </property>
    <property name="text">
     <string>To:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);



</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);



</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);


</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);


</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);



</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="moreabt1">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);




</string>
    </property>
    <property name="text">
     <string>More about this Event</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="Iris">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>190</y>
     <width>351</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-width: 0;
border-radius: 20px;
border-style: solid;
border-color: rgb(0,0,0,100);
color:black
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);


</string>
    </property>
    <property name="text">
     <string>Event Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);


</string>
    </property>
    <property name="text">
     <string>Venue:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);



</string>
    </property>
    <property name="text">
     <string>From:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);


</string>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);

font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);


</string>
    </property>
    <property name="text">
     <string>Date:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:white;color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);


</string>
    </property>
    <property name="text">
     <string>To:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);



</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);



</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);


</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);


</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 0;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);



</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="moreabt1_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);




</string>
    </property>
    <property name="text">
     <string>More about this Event</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="Mosaic">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>180</y>
     <width>351</width>
     <height>421</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-width: 0;
border-radius:20px;
border-style: solid;
border-color: rgb(0,0,0,100);
color:black
</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);


</string>
    </property>
    <property name="text">
     <string>Event Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);


</string>
    </property>
    <property name="text">
     <string>Venue:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);



</string>
    </property>
    <property name="text">
     <string>From:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);


</string>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_30">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);


</string>
    </property>
    <property name="text">
     <string>Date:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_31">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>61</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);


</string>
    </property>
    <property name="text">
     <string>To:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_32">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);



</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_33">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>80</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);



</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_34">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>130</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);


</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_35">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>240</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);


</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_36">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>181</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);



</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="moreabt1_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>12</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 0, 0);
font: 75 12pt &quot;Comic Sans MS&quot;;
border-width: 0;
border-radius: 10;
border-style: solid;
Background-color:rgb(0,0,0,0);
border-color: rgb(0,0,0,100);




</string>
    </property>
    <property name="text">
     <string>More about this Event</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_37">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>30</y>
     <width>821</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:black;
color:white;
font: 63 36pt &quot;Segoe UI Variable Text Semibold&quot;;</string>
   </property>
   <property name="text">
    <string>UPCOMING COLLEGE EVENTS</string>
   </property>
  </widget>
  <widget class="QPushButton" name="admin_log">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>700</y>
     <width>231</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Variable Text Semibold</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton#admin_log{background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);
border-radius:10px;
}
QPushButton#admin_log:hover{
background-color: rgb(75, 75, 75);
}
</string>
   </property>
   <property name="text">
    <string>Log Out</string>
   </property>
  </widget>
  <zorder>label_3</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>gallery_butt</zorder>
  <zorder>participation_butt</zorder>
  <zorder>contact_butt</zorder>
  <zorder>Ignitra</zorder>
  <zorder>Iris</zorder>
  <zorder>Mosaic</zorder>
  <zorder>label_37</zorder>
  <zorder>admin_log</zorder>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
